test/integration/golden_response/staging/query/statvar-obs.json

```diff
@@ -7,7 +7,7 @@
     {
       "cells": [
         {
-          "value": "dc/o/syrpc3m8q34z7"
+          "value": "dc/o/bqtfmc351v0f2"
```

Comment:
this diff would get stable after https://github.com/datacommonsorg/mixer/pull/467